local_node.sh

```diff
@@ -128,6 +128,11 @@ if [[ $overwrite == "y" || $overwrite == "Y" ]]; then
 	jq '.app_state["dogfood"]["val_set"][0]["power"]="5000"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
 	jq '.app_state["dogfood"]["last_total_power"]="5000"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
 
+	# x/exomint
+	# the default epoch_identifier is day, which means that rewards are minted once every day
```

Comment:
the comment said the default is **day**, but the code is **minute**.

Reply:
The comment talks about the default identifier (day) which the code changes (to minute)

Reply:
Will update the comment with some more clarity.